docs/reference/hooks.md

```diff
@@ -434,9 +434,9 @@ Hooks for customising the editing interface for pages and snippets.
 
 Rich text fields in Wagtail work with a list of 'feature' identifiers that determine which editing controls are available in the editor, and which elements are allowed in the output; for example, a rich text field defined as `RichTextField(features=['h2', 'h3', 'bold', 'italic', 'link'])` would allow headings, bold / italic formatting and links, but not (for example) bullet lists or images. The `register_rich_text_features` hook allows new feature identifiers to be defined - see [](rich_text_features) for details.
 
-(insert_editor_css)=
+(insert_global_admin_css)=
 
-### `insert_editor_css`
+### `insert_global_admin_css`
```

Comment:
Issue: `insert_global_admin_css` is already documented, there's no need to rename the existing one.

https://docs.wagtail.org/en/stable/reference/hooks.html#id21